tests/testSaturationCorrection.py

```diff
@@ -59,7 +57,6 @@ def testSaturation(self):
         )
 
         mask = maskedImage.getMask()
-        bitmaskBad = mask.getPlaneBitMask('BAD')
```

Comment:
This line (and the other removed one) was flagged as unused. @timj found the related commit.